core/sqf/sqenvcom.sh

```diff
@@ -40,12 +40,12 @@ export TRAFODION_VER="${TRAFODION_VER_MAJOR}.${TRAFODION_VER_MINOR}.${TRAFODION_
 export PRODUCT_COPYRIGHT_HEADER="2015-2017 Apache Software Foundation"
 ##############################################################
 # Trafodion authentication:
-#    Set TRAFODION_ENABLE_AUTHENTICATION to YES to enable
+#    Set TRAFODION_ENABLE_AUTHENTICATION=NO
 # authentication in the Trafodion environment.
 # LDAP configuration must also be setup--see
 # $TRAF_HOME/sql/scripts/traf_authentication_config for details.
 ##############################################################
-export TRAFODION_ENABLE_AUTHENTICATION=${TRAFODION_ENABLE_AUTHENTICATION:-NO}
+export TRAFODION_ENABLE_AUTHENTICATION=NO
```

Comment:
Was this an accidental commit for sqenvcom.sh where you disabled authentication to test your other changes?

export TRAFODION_ENABLE_AUTHENTICATION=${TRAFODION_ENABLE_AUTHENTICATION:-NO}  is what we need

Reply:
sorry, wrong for git add